.github/pull_request_template.md

```diff
@@ -26,6 +26,13 @@ You should include screenshots to help the reviewer understand what to look for
 changes are includes. GIFs help immensely when explaining interactions and animations.
 -->
 
+preReqs:
+
+[ ] following command succeeds `cd client && npm run checks && cd ../server && npm run checks`
```

Comment:
server side scripts, eslint config, etc. will come in a follow up PR
but doing here so i don't forget to add later